### ISSUE TITLE
Add h2 instead of h1 standard in the "Heading Block"

### ISSUE THREAD

**Issue by mbs1337** [soflyy/oxygen-bugs-and-features#1585]
**Describe the feature you'd like to see included in Oxygen.**
When adding "Heading" block trough Oxygen, the standard headline, should be H2 and not H1 as it is right now. If there is more than 1 H1 per page, it's very bad for SEO.

**What are the use cases for this feature?**
For SEO purpose. No more than 1 H1 per page is allowed

**Examples of this feature or functionality.**
I'm not "proud" of this, but Elementor got this function where if you add a header block, it's standard H2, and not H1. So if you think about SEO, this is the best way.

**Comment by Spellhammer** [comment]
You can change the h1 tag to h2-h6 in the Primary tab of the Properties Pane.

I do see how it would be much more convenient to have it default to h2.

**Comment by noodleyness** [comment]
Agreed! Constantly having to change to H2

**Comment by mbs1337** [comment]
> You can change the h1 tag to h2-h6 in the Primary tab of the Properties Pane.
> 
> I do see how it would be much more convenient to have it default to h2.

Yes I know I can change the headline type :) That is why I wan't to have the **standard/default** of H2, instead of H1

**Comment by jan-raap** [comment]
> You can change the h1 tag to h2-h6 in the Primary tab of the Properties Pane.
> 
> I do see how it would be much more convenient to have it default to h2.

fair enough, but since the tag selector is learnt the values should be made reactive in the ui and work together

<img width="267" height="255" alt="Image" src="https://github.com/user-attachments/assets/b5ced621-3610-460f-a181-9571ca99176e" />

see https://github.com/soflyy/oxygen-bugs-and-features/issues/3670 for more screenshots of the issue
